source/includes/_content_access_management.md

```diff
@@ -0,0 +1,283 @@
+# Content Access
+
+## Access Basics
+
+For each content item your organization uploads you will need coresponding access records. Access records define who can use the item for their ad campaigns and whether or not they have the ability to edit the item after it is in the system. Content Items have a one to many relationship to access records, although it is completely valid to have a content item only accessible by a single user.
+
+Access to a given content item can be defined at the following levels:
+- User based
+- Group Based
+- Group and Role based
+
+It's important to note that you you can create multiple access records tying content items to multiple 
+The ability to edit the content item can be granted by passing true for the `canEdit` flag. 
+
+## Get Access Records associated to a repository
+
+> Access Records Response
+
+```json
+{
+    "data": [
+        {
+            "contentItemId": "exampleId1",
+            "userId": "seattle_user_1",
+            "groupId": "seattle_office",
+            "canEdit": false,
+            "role": "group_user",
+            "isDeleted": false
+        }, // more records if applicable
```

Comment:
does the doc generated also barf on these comments? Github Markdown shows the same red highlight, probably because JSON doesn't support comments so this is invalid JSON.

Reply:
No - the doc generator handles it.